### ISSUE TITLE
Liquid warnings

### ISSUE THREAD

**Issue by sjengle** [vizsec/vizsec.github.io#13]
The `_data` files with the years first are causing some Liquid warnings:

```
Liquid Warning: Liquid syntax error (line 319): Expected id but found number in "member in site.data.2016-organizing_committee" in vizsec2016/index.html
Liquid Warning: Liquid syntax error (line 336): Expected id but found number in "member in site.data.2016-program_committee" in vizsec2016/index.html
Liquid Warning: Liquid syntax error (line 316): Expected id but found number in "group in site.data.2017-important_dates" in vizsec2017/index.html
Liquid Warning: Liquid syntax error (line 333): Expected id but found number in "member in site.data.2017-organizing_committee" in vizsec2017/index.html
Liquid Warning: Liquid syntax error (line 352): Expected id but found number in "member in site.data.2017-program_committee" in vizsec2017/index.html
Liquid Warning: Liquid syntax error (line 367): Expected id but found number in "supporter in site.data.2017-supporters" in vizsec2017/index.html
```

I tested changing the filenames to start with a letter versus a number (e.g. `y2017-important_dates.yml` versus `2017-important_dates.yml`), and it seemed to resolve the problem. But, everything seems to still work even with those warnings.

**Comment by jgoodall** [comment]
What version of `jekyll` are you using? I have 3.0.5 and am not getting any warnings.

**Comment by sjengle** [comment]
I'm using the version that comes with `bundle`, which is 3.4.3. That is the version of Jekyll Github pages is running currently:

https://pages.github.com/versions/

**Comment by jgoodall** [comment]
I had older version of jekyll. confirmed issue and fixed - f9bc47074f21d6e9410e754d690c3039bba63d92
